app/com/linkedin/drelephant/tuning/Manager.java

```diff
@@ -0,0 +1,13 @@
+package com.linkedin.drelephant.tuning;
+
+public interface Manager {
```

Comment:
Please add comments for the class, and use Javadoc style comments.

Reply:
done